Client/Application/AppDelegate.swift

```diff
@@ -260,22 +260,35 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         var url: String?
         var isPrivate: Bool = false
+        var fxaParams: [String: String] = [
+            "view" : "",
+            "email" : "",
+            "code" : ""
+        ]
         for item in (components.queryItems ?? []) as [NSURLQueryItem] {
             switch item.name {
             case "url":
                 url = item.value
             case "private":
                 isPrivate = NSString(string: item.value ?? "false").boolValue
+            case "fxa":
```

Comment:
I know this loop/switch is already here. But using url.getQuery() will return a dictionary and prevent you from having to loop through results like this.